bounties_api/bounties/utils.py

```diff
@@ -86,3 +86,8 @@ def bounty_url_for(bounty_id, platform=None):
         base_url = settings.PLATFORM_MAPPING[platform]
     url = '{}/bounty/v1/{}/'.format(base_url, bounty_id)
     return url
+
+def profile_url_for(public_address):
```

Comment:
@c-o-l-o-r actually you need the mapping logic here. Why don't we have the above function produce a root url - then have profile_url and bounty_url funcs use that

Reply:
good call, I updated it here b63f50e